include/oneapi/dpl/pstl/hetero/numeric_impl_hetero.h

```diff
@@ -155,6 +155,22 @@ __pattern_transform_scan_base_impl(_ExecutionPolicy&& __exec, _Iterator1 __first
         .wait();
 }
 
+template <typename _Iterator1, typename _Iterator2>
+constexpr bool
+__is_equal_iterators(_Iterator1 it1, _Iterator2 it2)
```

Comment:
As far as we have a plural form  - "iterators" we have to use the other form of "to be" verb  or don't use "to be" at all. So, my suggestion
`__are_equal_iterators` or `__check_equal_iterators`

Reply:
OK, let's use your variant ``` __check_equal_iterators```

Reply:
Fixed.